sascha/iban/script.js

```diff
@@ -0,0 +1,176 @@
+//sets the bootstrap element for the success message
+    function successMessage(string){
+        document.getElementById("copyBox").className = "alert alert-success";
+        document.getElementById("copyBox").innerHTML = string;
+    }
+	
+
+//This is the function for the entire iban calculator program
+function ibanCalculator() {
+    //global variable declarations
+    var landCode = 'NL';
+    var bankRekening = document.getElementById("bankRekening").value;
+    var bankCode = document.getElementById("bankName").value;
+    var controleGetal;
+    var ibanMathString;
+	
+    /*Function Expressions*/
+    //this function resets the warning box
+    var resetWarningBox = function(){

```

Comment:
Dit is een functie-expressie, en in tegenstelling tot functie-declaraties hoor je deze af te sluiten met een ; (op regel 24).

Voor een oefening is het prima om de verschillende manieren om functies te definiëren door elkaar te gebruiken, maar in productie-code zou ik 1 manier kiezen.